buildSrc/version.properties

```diff
@@ -1,5 +1,5 @@
-elasticsearch     = 5.0.0-alpha3
-lucene            = 6.0.0
+elasticsearch     = 5.0.0
```

Comment:
Yes, I needed to add the 5.0.0 version back since alpha3 is being released with Lucene 6.0.0. I will merge this PR after alpha3 is out.